### ISSUE TITLE
WRONG RTON number header hex 0x49

### ISSUE THREAD

**Issue by SMFDrummer** [oobol2000oo/RETON#22]
Please change it to 0x45 instead in CHUNK_TYPE
0x49 is deprecated

**Comment by nineteendo** [comment]
Agreed, `0x49` would be `varuint64_t` (a signed unsigned int64), which doesn't make sense:

Bytecode | Type | Limitations
-------- | ---- | -----------
0x24 | uvarint32_t | 0-2147483647
0x25 | varint32_t | -2147483648-2147483647
0x28 | uvaruint32_t | 0-4294967295
0x44 | uvarint64_t | 0-9223372036854775807
0x45 | varint64_t | -9223372036854775808-9223372036854775807
0x48 | uvaruint64_t | 0-18446744073709551615

**Comment by SMFDrummer** [comment]
I am trying to rewrite the RETON project using Kotlin, but besides 0x49, I have also encountered some decoding issues with rton2json. For example, I am unable to correctly transcode and shape data because there are too many types of data. I can provide the source code of the semi-finished Kotlin project. Can you give me some improvement suggestions?
